scripts/install

```diff
@@ -133,34 +61,33 @@ ynh_script_progression --message="Configuring NGINX web server..."
 # Create a dedicated NGINX config
 ynh_add_nginx_config
 
+# Create a dedicated systemd config
+ynh_add_systemd_config
+
 #=================================================
 # SPECIFIC SETUP
 #=================================================
 # ADD A CONFIGURATION
 #=================================================
 ynh_script_progression --message="Adding a configuration file..."
 
-ynh_add_config --template="../conf/db.example.inc.php" --destination="$final_path/config/db.inc.php"
-
-chmod 400 "$final_path/config/db.inc.php"
-chown $app:$app "$final_path/config/db.inc.php"
-
-#=================================================
-# SETUP SYSTEMD
-#=================================================
-ynh_script_progression --message="Configuring a systemd service..."
+ynh_add_config --template="../conf/db.example.inc.php" --destination="$install_dir/config/db.inc.php"
 
-# Create a dedicated systemd config
-ynh_add_systemd_config
+chmod 400 "$install_dir/config/db.inc.php"
+chown $app:$app "$install_dir/config/db.inc.php"
```

Comment:
This is now replaced by a unique .env file https://github.com/movim/movim/blob/master/INSTALL.md#2-dotenv-configuration :)